Slim/Control/Commands.pm

```diff
@@ -381,7 +381,7 @@ sub clientForgetCommand {
 	# player that's still playing the same way as a player that's turned off while still playing, so
 	# we can make it start playing again if it reappears and the user told us to resume playing
 	# when powering on.
-	$client->persistPlaybackStateForPowerOff();
+	$prefs->client($client)->set('playingAtPowerOff', $client->controller()->isPlaying(1)) if $client->power();
```

Comment:
Ahm... to many `client` mentions on a single line for my taste 😁. This doesn't change anything in the behavior from the previous implementation, does it? Could you please at least leave it in a structured way to make it easier on the eye?

Reply:
:smile: you know me, and I know you :smile:

Reply:
Updated to try a compromise :rofl: